### ISSUE TITLE
[BUG] Did not get too short pop up when I ended an indoor workout with 23 steps

### ISSUE THREAD

**Issue by kanakagr** [Share-Impact/Android-Impact-Issues#78]
**Describe the bug**
Did not get too short pop up when I ended an indoor workout with 23 steps

**To Reproduce**
1. Start an indoor Workout
2. Try to end it with less than 125 steps
3. Couldn't reproduce it though. Happened only once

**Expected behavior**
Too short pop up should come

**Screenshots**
No screenshots or recording

**Version [Check app's setting screen]:**
2.15.0

**Additional context**

**Comment by AnujaPednekar** [comment]
Checked in code, not able to find any flaw

**Comment by AnujaPednekar** [comment]
@kanakagr Please send a screenshot of your workout history of 18th May

**Comment by AnujaPednekar** [comment]
![Screenshot_20200520-115334](https://user-images.githubusercontent.com/43374048/82413267-ac02f880-9a92-11ea-82f0-42dcde88f5ef.png)

**Comment by AnujaPednekar** [comment]
![Screenshot_20200520-115334](https://user-images.githubusercontent.com/43374048/82414516-c938c680-9a94-11ea-8b3a-5c1e89a8b270.png)

I checked your GA events for workouts done on 18th May between 5 to 6 pm, I can see you got too short pop up at appropriate times but i spotted one issue which was in the workout done at 5:43 pm, you had covered a distance 0.0915 and you got too short pop up but the workout was also created. Is it suppose to happen @parthanjaria ?

**Comment by AnujaPednekar** [comment]
![WhatsApp Image 2020-05-20 at 12 23 12 PM](https://user-images.githubusercontent.com/43374048/82415309-f46fe580-9a95-11ea-8777-95e00280384a.jpeg)

I checked your GA events for workouts done on 18th May between 5 to 6 pm, I can see you got too short pop up at appropriate times but i spotted one issue which was in the workout done at 5:43 pm, you had covered a distance 0.0915 and you got too short pop up but the workout was also created. Is it suppose to happen @parthanjaria ?

**Comment by parthanjaria** [comment]
No if the user gets too short popup then the workout is not supposed to be created. if it has, it is a bug.

**Comment by AnujaPednekar** [comment]
Closing this issue since too short pop up appeared at appropriate times. Creating another issue since u didn't get to see TY screen. 

Please reopen the issue if u face again.

**Comment by sagrgpt** [comment]
Check issue #92 for more details.
